client/state/sites/plans/test/reducer.js

```diff
@@ -6,70 +6,308 @@ import { expect } from 'chai';
 /**
  * Internal dependencies
  */
-import { FETCH_SITE_PLANS, REMOVE_SITE_PLANS } from 'state/action-types';
-import { initialSiteState, plans } from '../reducer';
+import {
+	SITE_PLANS_FETCH,
+	SITE_PLANS_FETCH_COMPLETED,
+	SITE_PLANS_FETCH_FAILED,
+	SITE_PLANS_TRIAL_CANCEL,
+	SITE_PLANS_TRIAL_CANCEL_FAILED,
+	SITE_PLANS_TRIAL_CANCEL_COMPLETED,
+	SITE_PLANS_REMOVE
+} from 'state/action-types';
+import { plans } from '../reducer';
 
 describe( 'reducer', () => {
 	describe( '#plans()', () => {
-		it( 'should default to an empty object', () => {
+		it( 'should return an empty state when original state is undefined and action is empty', () => {
 			const state = plans( undefined, {} );
 
 			expect( state ).to.eql( {} );
 		} );
 
-		it( 'should index plans by site ID', () => {
-			const siteId = 11111111,
-				state = plans( undefined, {
-					type: FETCH_SITE_PLANS,
-					siteId: siteId
+		it( 'should return an empty state when original state and action are empty', () => {
+			const original = Object.freeze( {} ),
```

Comment:
Good idea to wrap the initial state in `Object.freeze` in all of these tests. :+1: